multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -38,10 +38,12 @@
 OUTPUT = "output"
 WEIGHT = "weight"
 FEATURES = "features"
+RAW_FEATURES = "raw_features"
 MASKS = "masks"
 PROBABILITY = "probability"
 COLUMN_FEATURES = "column_features"
 BBOX = "bbox"
+ROIS = "rois"
```

Comment:
Is `rois` the plural form of `roi`?

Reply:
Yes, the key name "rois" is inherited from gluon cv: https://github.com/dmlc/gluon-cv/blob/f662daeecccd2ec6b8978ccee1d40c1d0f03ceb5/gluoncv/auto/data/dataset.py#L611.